lib/kennel/models/monitor.rb

```diff
@@ -35,6 +35,7 @@ class Monitor < Base
         notify_no_data: -> { true },
         no_data_timeframe: -> { notify_no_data ? 60 : nil },
         notify_audit: -> { true },
+        new_host_delay: -> { 300 },
```

Comment:
should pull from MONITOR_OPTION_DEFAULTS too ? (so we don't have to store the default value everywhere)